widgets/widgets.go

```diff
@@ -74,13 +77,14 @@ func BaseWidget(style, inputType string) *Widget {
 		formcommon.HIDDEN,
 		formcommon.IMAGE,
 		formcommon.MONTH:
-		urls = append(urls, formcommon.CreateUrl("templates/%s/input.html"))
+		urls = append(urls, filepath.Join(templatesPrefix, "input.html"))
 	default:
-		urls = append(urls, formcommon.CreateUrl("templates/%s/input.html"))
+		urls = append(urls, filepath.Join(templatesPrefix, "input.html"))
```

Comment:
Are you sure you should .Join with templatesPrefix here?

Reply:
Yeah, fixed. Thanks for noticing! (and the above too)